lib/discovery_api_web/controllers/dataset_download_controller.ex

```diff
@@ -1,6 +1,6 @@
 defmodule DiscoveryApiWeb.DatasetDownloadController do
   use DiscoveryApiWeb, :controller
-  alias DiscoveryApiWeb.DatasetMetricsService
+  alias DiscoveryApiWeb.Services.MetricsService
```

Comment:
This is a total nitpick, but if this module is namespaced under Services, I'd prefer to just call it Metrics.
```suggestion
  alias DiscoveryApiWeb.Services.Metrics
```

Reply:
I generally agree. I think the main thing was to make it read better when aliased. We could possibly just alias Services and then call it as Service.Metrics. We'll try that and see how it feels.

Reply:
That makes sense. I just assumed one type of 'service' in a module, but I could see how it would be unclear with multiple. Sounds good.